#!/usr/bin/env python
PACKAGE = "quadrotor_dynamics"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
 
gen.add("WindX", double_t, 0, "Wind in X-Richtung", 0.0,  -2.0, 2.0)
gen.add("WindY", double_t, 0, "Wind in Y-Richtung", 0.0,  -2.0, 2.0)
gen.add("WindZ", double_t, 0, "Wind in Z-Richtung", 0.0,  -2.0, 2.0)
 
exit(gen.generate(PACKAGE, "quadrotor_dynamics", "Wind"))
